.github/workflows/js/publish.yaml

```diff
@@ -0,0 +1,90 @@
+name: Lint, Test, Build and Publish to GitHub Package Registry
```

Comment:
Can we make mage smart enough to just know when to run the publish step? Then we can only have one mage workflow.